network/betweenness.go

```diff
@@ -95,12 +95,12 @@ func Betweenness(g graph.Graph) map[int]float64 {
 	return cb
 }
 
-// BetweennessWeighted returns the non-zero etweenness centrality for nodes in the weighted
+// BetweennessWeighted returns the non-zero betweenness centrality for nodes in the weighted
 // graph g used to construct the given shortest paths.
 //
 //  C_B(v) = \sum_{s ≠ v ≠ t ∈ V} (\sigma_{st}(v) / \sigma_{st})
 //
-// where \sigma_{st} and \sigma_{st}(v) are number of the shortest paths from s to t.
+// where \sigma_{st} and \sigma_{st}(v) are the number of shortest paths from s to t.
```

Comment:
Good thing, since I had to go back and look - finding the same word ordering error in the first comment too.